src/chunk_constraint.c

```diff
@@ -159,6 +159,8 @@ chunk_constraint_insert_relation(Relation rel, ChunkConstraint *constraint)
 	bool		nulls[Natts_chunk_constraint] = {false};
 	NameData	constraint_name;
 
+	/* quiet valgrind */
+	memset(constraint_name.data, 0,  NAMEDATALEN);
```

Comment:
Again, it is clear from the line below that this initialization is not strictly needed. Shouldn't this be a suppression instead? Maybe not a big deal, but just pointing out that this isn't a bug and comes at a cost.

Reply:
Yes, this is not a bug. Don't know what's best to do.